cmd/tailscale/cli/cli_test.go

```diff
@@ -489,18 +490,20 @@ func TestCheckForAccidentalSettingReverts(t *testing.T) {
 			flagSet := newUpFlagSet(goos, &upArgs)
 			flags := CleanUpArgs(tt.flags)
 			flagSet.Parse(flags)
+			env := upCheckEnv{
```

Comment:
move this down a bit to just before `applyImplicitPrefs`. As written this makes it look like `prefsFromUpArgs` needs this.